org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/SbtInputs.scala

```diff
@@ -143,14 +144,36 @@ class SbtInputs(installation: IScalaInstallation,
     store.compilerBridgeFor(installation)(javaMonitor.newChild(10)).right.map {
       compilerBridge =>
         // prevent zinc from adding things to the (boot)classpath
-        val cpOptions = new ClasspathOptions(false, false, false, /* autoBoot = */ false, /* filterLibrary = */ false)
+        val cpOptions = ClasspathOptions.create(false, false, false, /* autoBoot = */ false, /* filterLibrary = */ false)
         Compilers(
-          new AnalyzingCompiler(scalaInstance, CompilerBridgeProvider.constant(compilerBridge.toFile), cpOptions, _ ⇒ (), None),
+          new AnalyzingCompiler(scalaInstance,
+              new CompilerBridgeProvider {
+                def fetchCompiledBridge(si: ScalaInstance, logger: Logger) = si match {
+                  case scalaInstance => compilerBridge.toFile
```

Comment:
the match is not required here because `scalaInstance` is not used

Reply:
actually I dont remember why there is not just `def fetchCompiledBridge(si: ...) = complierBridge.toFile`?